js/main.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+function start(canvas) {
+  const game = new Game(canvas.getGridDimensions().width, canvas.getGridDimensions().height);
+
+  game.init();
+  game.draw(canvas);
+
+  function frame() {
+    canvas.clear();
+    game.update();
+    game.draw(canvas);
+
+    if (game.gameOver) {
+      clearTimeout(setIntervalId);
```

Comment:
You shouldn't stop the game loop when the game is over, because the player might want to start a new game, and the game must continue to update / redraw.
The game loop will run indefinitely.